.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -48,6 +48,11 @@ on:
           Note: it doesn't affect ./devops/actions/run-tests/* as these actions
           call checkout again and therefore override the devops directory, so
           configs/dependecies from input.ref are used.
+      sycl_cts_ref:
+        description: "Commit SHA or branch to checkout"
+        type: string
+        default: "main"
+        required: False
```

Comment:
Can we have more opaque parameters like `devops_ref` and `tests_ref` so that all the actions would use the same name at the high-level entry point?

Reply:
It this what you mean? [bac35ea](https://github.com/intel/llvm/pull/17074/commits/bac35ea74b986dd9360c923d06f4be2dcb7d7294)